docs/CHANGELOG.asciidoc

```diff
@@ -44,6 +45,77 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.10.1]]
+== {kib} 7.10.1
+
+For detailed information about the 7.10.1 release, review the following enhancements and bug fixes.
+
+For the breaking changes, refer to the <<breaking-changes-7.10,breaking changes in 7.10>>.
+
+[float]
+[[enhancement-v7.10.1]]
+=== Enhancements
+Machine Learning::
+* Adds unsigned_long support to data frame analytics and anomaly detection {kibana-pull}82636[#82636]
+Platform::
+* Fixes a bug causing searching for saved objects using special characters such as * or - to not return any results {kibana-pull}82693[#82693]
+
+[float]
+[[bug-v7.10.1]]
+=== Bug fixes
+APM::
+* Fixes missing `service.node.name` {kibana-pull}84269[#84269]
+* Page load chart breakdown tooltip formatting {kibana-pull}83627[#83627]
+Canvas::
+* Fixes elements not being updated properly when filter is changed on workpad {kibana-pull}81863[#81863]
+Dashboard::
+* Makes sure `to` is 'now' and not the same as `from` {kibana-pull}81524[#81524]
```

Comment:
There was no release notes on this PR, but I think we should change the wording here to:

> Fixes an issue when exporting a Saved Search Visualization (inside of a Dashboard) to CSV was returning blank rows.